src/test/java/pl/damianszczepanik/jenkins/buildhistorymanager/descriptors/actions/DeleteArtifactsMatchingPatternsActionDescriptorTest.java

```diff
@@ -0,0 +1,29 @@
+package pl.damianszczepanik.jenkins.buildhistorymanager.descriptors.actions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import org.jenkinsci.plugins.structs.SymbolLookup;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class DeleteArtifactsMatchingPatternsActionDescriptorTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
j like jenkins or manybe r like rule?

Reply:
> j like jenkins or manybe r like rule?

Is this a request to rename the variable from `j` to `r` or to `jr` or something else???

Reply:
I followed the code style used in all descriptor test classes, where Jenkins Rule is defined as `public JenkinsRule j = new JenkinsRule();` 
For example: https://github.com/jenkinsci/build-history-manager-plugin/blob/master/src/test/java/pl/damianszczepanik/jenkins/buildhistorymanager/descriptors/actions/DeleteBuildActionDescriptorTest.java#L18

Reply:
Right, leave it as it is!